packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { themeColor, themeSpacing } from '../../utils/themeUtils'
+
+export default styled.h1`
+  max-width: 620px;
+  margin: ${themeSpacing(6, 0, 8)};
+  padding-bottom: 6px;
+  border-bottom: 1px solid ${themeColor('tint', 'level5')};
+  color: ${themeColor('tint', 'level5')};
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 18px;
+  -webkit-font-smoothing: auto;
```

Comment:
Why reset the `-webkit-font-smoothing` here?

Reply:
In `GlobalStyle.ts` the style rule `-webkit-font-smoothing: antialiased;` is set on the `body`. This makes the `font-weight: 700` does not look strong enough.

`-webkit-font-smoothing: antialiased;`
![image](https://user-images.githubusercontent.com/7781865/80078363-1afe3780-854f-11ea-9848-9b9536868862.png)

`-webkit-font-smoothing: auto;` (overwrite)
![image](https://user-images.githubusercontent.com/7781865/80078337-0fab0c00-854f-11ea-8fe6-ff504eefe52b.png)

That's why I overwrite `antialiased`

Reply:
If this is a problem, let's remove the global one instead. I'd rather have font rendering consistent where possible.

Reply:
I've asked @royderks and @chrisvanmook in the #frontend channel

Reply:
Removed it